website/src/content/docs/docs/howtos/Generate client libraries/06types.mdx

```diff
@@ -6,6 +6,166 @@ import { ClientTabs, ClientTabItem } from "@components/client-tabs";
 
 This page documents what type definitions in TypeSpec are generated as in emitted libraries
 
+## Namespace
+
+The namespace for models, enums, and unions will follow the namespace they belong to. You can use `@clientNamespace` to override it if needed.
+
+### Model in namespace
+
+<ClientTabs>
+<ClientTabItem lang="typespec" >
+
+```tsp
+namespace MyService;
+
+model Foo {
+  prop: string;
+}
+```
+
+</ClientTabItem>
+
+<ClientTabItem lang="python">
+
+```python
+from my_service.models import Foo
+
+foo = Foo(prop="hello")
+```
+
+</ClientTabItem>
+<ClientTabItem lang="csharp" >
+
+```csharp
+// TODO
+```
+
+</ClientTabItem>
+<ClientTabItem lang="typescript" >
+
+```typescript
+// TODO
+```
+
+</ClientTabItem>
+<ClientTabItem lang="java" >
+
+As `flavor=azure`, Java emitter generates the model classes under `models` package.
+
+```java
+import myservice.models.Foo;
```

Comment:
cspell fails on this `myservice`.

Maybe just call the namespace `Service` in source?